src/assets/misc/connectingChains/connectingChains.asset.ts

```diff
@@ -0,0 +1,107 @@
+import { ItemInteractionType } from 'pandora-common';
+
+DefineAsset({
+	name: 'Connecting Chains',
+	size: 'small',
+	graphics: 'graphics.json',
+	colorization: {
+		Chains: {
+			name: 'Chains',
+			default: '#DDDDDD',
+		},
+	},
+	// size:400, y:550, centered
+	preview: 'preview.png',
+	modules: {
+		lock: {
+			type: 'lockSlot',
+			name: 'Lock',
+			occupiedProperties: {
+				blockAddRemove: true,
+			},
+		},
+		chains: {
+			type: 'typed',
+			name: 'Connecting Chains',
+			interactionType: ItemInteractionType.ADD_REMOVE,
+			variants: [
+				{
+					id: 'none',
+					name: 'No Chain',
+					default: true,
+				},
+				{
+					id: 'beltChain',
+					name: 'Belt Chains',
+					switchMessage: 'SOURCE_CHARACTER connects the cuffs to TARGET_CHARACTER_DYNAMIC_POSSESSIVE belt with short chains.',
+					properties: {
+						poseLimits: {
+							arms: {
+								position: 'back',
+							},
+							bones: {
+								arm_l: 70,
+								elbow_l: 20,
+								arm_r: 70,
+								elbow_r: 20,
+							},
+						},
+						attributes: {
+							provides: [
+								'Restraint',
+								'Restraint_arms',
+							],
+							requires: [
+								'Wrist_cuffs_chainable',
+								'Belt_chainable',
+							],
+						},
+					},
+				},
+				{
+					id: 'armbinderChain',
+					name: 'Kneeling Chains',
+					switchMessage: 'SOURCE_CHARACTER connects the cuffs to TARGET_CHARACTER_DYNAMIC_POSSESSIVE armbinder with short chains.',
+					properties: {
+						poseLimits: {
+							legs: 'kneeling',
+							bones:
+								{
+									leg_l: [[-28, 5]],
+									leg_r: [[-28, 5]],
+								},
+						},
+						attributes: {
+							provides: [
+								'Restraint',
+								'Restraint_legs',
+							],
+							requires: [
+								'Ankle_cuffs_chainable',
+								'Armbinder_chainable',
+							],
+						},
+					},
+				},
+			],
+		},
+	},
+	chat: {
```

Comment:
These messages are no longer correct. 

You could also remove the 'chat' property completely, then default neutral asset add/remove messages are used, plus your switch messages from the module changes.